ale/drivers/mes_drivers.py

```diff
@@ -225,6 +225,21 @@ class MessengerMdisIsisLabelNaifSpiceDriver(IsisLabel, NaifSpice, Framer, Driver
     Driver for reading MDIS ISIS3 Labels. These are Labels that have been ingested
     into ISIS from PDS EDR images but have not been spiceinit'd yet.
     """
+    @property
+    def platform_name(self):
+        """
+        Returns the name of the platform containing the sensor. This is usually
+        the spacecraft name.
+
+        Messenger MDIS ISIS labels use upper camel case so this converts it to\
+        all upper case.
+
+        Returns
+        -------
+        : str
+          Spacecraft name
+        """
+        return super().platform_name.upper()
```

Comment:
I thought about making this change in the IsisLabel class but I'm not sure if that's correct

Reply:
I'm not sure either. Here definitely seems valid, though.